components/gitpod-db/src/typeorm/migration/1679909342076-AddOrganizationSettings.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddOrganizationSettings1679909342076 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "CREATE TABLE IF NOT EXISTS d_b_team_settings (  teamId int(11) DEFAULT NULL,  disabledWorkspaceSharing tinyint(4) NOT NULL DEFAULT '0', PRIMARY KEY (teamId)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;",
```

Comment:
```suggestion
            "CREATE TABLE IF NOT EXISTS d_b_org_settings (  teamId int(11) DEFAULT NULL,  disabledWorkspaceSharing tinyint(4) NOT NULL DEFAULT '0', PRIMARY KEY (teamId)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;",
```

I'm unsure about the `teamId` type of `int(11)`, normally these are UUIDs as varchar

Reply:
Ye, it's changed to var(36)

Reply:
Also, I think you'll want to also have a secondary index on (teamId, _delete) as most direct lookups will need to check the deleted value

Reply:
If `org` is soft deleted first, then I will add `id` `_delete` field to this table